openbsd/OpenBSDMachine.c

```diff
@@ -116,7 +116,7 @@ Machine* Machine_new(UsersTable* usersTable, uid_t userId) {
 
    this->cpuSpeed = -1;
 
-   return this;
+   return super;
```

Comment:
@BenBE I'm not sure if this change is correct, let me know your thoughts. I can't remember why I changed `this` to `super`. This is likely my last concern before this gets merged, but I'm going to retest the pr on my end like you asked as we speak.

Reply:
That change is correct, as `this` is `OpenBSDMachine*` and `super` is `Machine*`. If you look at the return type you see it match `Machine*`, thus `super`. With `&this->super == (Machine*)this`, that doesn't make much difference from a "value" perspective, but as `Machine* super = &this->super;` gives this value in a type-safe manner. I hope this explains this.

Also note: When I updated this PR earlier, I rechecked the various changes from a type perspective to make change.